systemd/systemd.go

```diff
@@ -186,6 +192,26 @@ func NewCollector(logger log.Logger) (*Collector, error) {
 		"Service unit egress IP accounting in packets.",
 		[]string{"name"}, nil,
 	)
+	watchdogEnabled := prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "watchdog_enabled"),
```

Comment:
Should we also declare a subsystem here?

```suggestion
const watchdogSubsystem = "watchdog"
...

		prometheus.BuildFQName(namespace, watchdogSubsystem, "enabled"),`
```